applications/h264/h264_endoscopy_tool_tracking_distributed/cpp/viz_fragment.cc

```diff
@@ -0,0 +1,45 @@
+/*
+ * SPDX-FileCopyrightText: Copyright (c) 2024 NVIDIA CORPORATION & AFFILIATES. All rights
+ * reserved. SPDX-License-Identifier: Apache-2.0
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef VIZ_FRAGMENT_CC
+#define VIZ_FRAGMENT_CC
+
+#include <holoscan/holoscan.hpp>
+#include <holoscan/operators/holoviz/holoviz.hpp>
+
+class VizFragment : public holoscan::Fragment {
+ private:
+  uint32_t width_ = 0;
+  uint32_t height_ = 0;
+
+ public:
+  VizFragment(const uint32_t width, const uint32_t height) : width_(width), height_(height) {}
+
+  void compose() override {
+    std::shared_ptr<UnboundedAllocator> visualizer_allocator;
```

Comment:
I don't see this variable initialized with `make_resource<T>()` here, so it might behave as if we're passing a null pointer.

The allocator is used when headless mode or the render buffer is used.

```
  spec.param(
      allocator_, "allocator", "Allocator", "Allocator used to allocate render buffer output.");
```

So, I don't think this parameter affects the application.

Could we remove this line and line 40?